### DISCUSSION TITLE
Other Repos [where issues were turned off] should link to this Repo for Feedback

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#48] [General]
It looks like issues were turned off for most(/all?) repos in the BattlesnakeOfficial account.

However it doesn't look like all of the repos have anything that points to this feedback repo!
For instance the board doesn't have any mention of this repo, https://github.com/BattlesnakeOfficial/board.

Maybe an addition to those READMEs would help people find this discussion repo?

**Comment by andrewdmaclean** [comment]
Great point! I'll add this to my todo list for today
